test/items_formatter_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule ItemsFormatterTest do
+  use ExUnit.Case
+  doctest ToDo
+
+  test "to do menu options displayed with no items" do
+    assert ToDo.ItemsFormatter.items_for_display([ 'Go to yoga tonight', 'make to do list app']) ==
+    "To Do List: \n[1] Go to yoga tonight\n[2] make to do list app"
```

Comment:
This line is too long and is difficult to read. Try and pull some things into temp variables so that the assertion reads better.